int8 device
int8 valve
---
string status